webapp/src/components/app-bar-icon/app-bar-icon.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {useSelector} from 'react-redux';
+
+import {getPluginURL} from '../../selectors';
+
+const appBarIconPath = '/public/app-bar-icon.png';
+
+export default function AppBarIcon() {
+    const pluginURL = useSelector(getPluginURL);
+    const iconURL = pluginURL + appBarIconPath;
+
+    return (
+        <img src={iconURL}/>
```

Comment:
Note that in the app bar meeting we decided we should instead just have the plugin provide an image url to the webapp instead of this react component

Reply:
If the point is to make it simpler and let the webapp handle it, why not passing only the iconURL instead of a component?

I see potential value in allowing a component (like conditional rendering) but not sure if we want to give the level of freedom.

Reply:
> If the point is to make it simpler and let the webapp handle it, why not passing only the iconURL instead of a component?

@larkox This is the plan now, please see my comment above 🙂

Reply:
I still don't get the potential improvement of passing the component. Could you clarify?

My point is, why do this:
`registry.registerAppBarComponent(<img src='url'>, ...)`
when we can do directly:
`registry.registerAppBarComponent(url, ...)`

That way there is no chance for the plugin to create weird objects in the App Bar, and leaves all the implementation details to the webapp.

Reply:
@larkox The code above is outdated. We are going to pass the icon URL instead of a react component